newsletter2go/views/js/nl2go_script.js

```diff
@@ -45,20 +47,37 @@ window.addEventListener('load', function () {
         window.open(baseUrl + '?' + $.param(params), '_blank');
     });
 
+    disconnect.addEventListener('click', function () {
+        var xmlHttp = new XMLHttpRequest(),
+            parameters = 'token=token&ajax=a&tab=Newsletter2GoTab&action=disconnect';
```

Comment:
I am confused by these parameters and not sure if we need all them, like 'token', ajax